onnx/defs/nn/defs.cc

```diff
@@ -266,6 +267,10 @@ and computes the output.)DOC";
             schema.Attr("output_shape",
                         "The shape of the output.",
                         AttributeProto::INTS, OPTIONAL);
+            schema.Attr("adjs",
```

Comment:
feel like the name "output_padding" makes more sense, no?

Reply:
Both Caffe2 and MXNet use adjs, only PyTorch uses output_padding. I try to a more widely used name. I agree with that output_padding is more straightforward.

Reply:
I'm partial to `output_padding`, but I'm also a PyTorch developer :)